src/moj/components/cookie-banner/template.njk

```diff
@@ -0,0 +1,24 @@
+{%- from "govuk/components/button/macro.njk" import govukButton %}
+
+<div class="cookie-banner govuk-width-container cookie-banner--visible" role="region" aria-label="cookie banner">
```

Comment:
```suggestion
<div class="cookie-banner govuk-width-container cookie-banner--show" role="region" aria-label="cookie banner">
```

I think the class should be `cookie-banner--show`? Was also wondering if this should be configurable by an argument?

Reply:
```suggestion
<div class="moj-cookie-banner govuk-width-container moj-cookie-banner--show" role="region" aria-label="cookie banner">
```
Ah, also all the classes need the `moj-` prefix.

Reply:
Great catch, thanks, these have been updated